### PR TITLE
Updated MBC1 & About Section

### PR DESCRIPTION
**Pull request by wawesomeNOGUI**
I fixed some grammar, and made the first paragraph more concise in MBC1. I also addressed issue #65 by adding a sentence about what the Pan Docs is for in the About section.

### CODE REVIEW

**Comment by wawesomeNOGUI** [comment]
I also addressed issue #65 by adding a sentence about what the Pan Docs is for

File: content/MBC1.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-MBC chip to another - or even to make it compatible to several different
+This is the first MBC chip for the Game Boy. Any newer MBC chips 
+work similarly, so it is relatively easy to upgrade a program from one
+MBC chip to another - or to make it compatible with several different
 types of MBCs.
 
-Note that the memory in range 0000-7FFF is used for both reading from
-ROM, and for writing to the MBCs Control Registers.
+Note that the memory in range 0000-7FFF is used both for reading from
+ROM, and writing to the MBCs Control Registers.
```
Comment (aaaaaa123456789) [inline content/MBC1.md:9]: No comma before the preposition in a list of two items

File: content/MBC1.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ higher (01/21/41/61).
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
-External RAM is often battery buffered, allowing to store game positions
-or high score tables, even if the Game Boy is turned off, or if the
+External RAM is often battery buffered to store game positions
```
Comment (aaaaaa123456789) [inline content/MBC1.md:29]: Removing the comma and merging the clauses makes them unclear. "...battery-buffered, which allows it to store..." would be better.

File: content/MBC1.md [hunk 3]
```diff
@@ -26,8 +26,7 @@ higher (01/21/41/61).
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
-External RAM is often battery buffered to store game positions
-or high score tables while the Game Boy is turned off, or if the
+External RAM is often battery buffered, allowing for the storage of game memory while the Game Boy is turned off, or if the
```
Comment (wawesomeNOGUI) [inline content/MBC1.md:29]: Maybe we could shorten the battery backed RAM part to this?

Comment (aaaaaa123456789) [reply content/MBC1.md:29]: Memory isn't stored; memory is the location where data is stored. "Game memory" should be "game data".

File: content/MBC1.md [hunk 4]
```diff
@@ -26,8 +26,7 @@ higher (01/21/41/61).
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
-External RAM is often battery buffered, allowing to store game positions
-or high score tables, even if the Game Boy is turned off, or if the
+External RAM is often battery buffered, allowing for the storage of game data while the Game Boy is turned off, or if the
```
Comment (wawesomeNOGUI) [inline content/MBC1.md:29]: Cool, edited that line, data does make more sense than memory.

File: content/MBC1.md (hunk 1 above)
Comment (wawesomeNOGUI) [reply content/MBC1.md:9]: Do I have to edit that in, or did you already change it?

**Comment by avivace** [comment]
Thanks @aaaaaa123456789 and @wawesomeNOGUI ! Did some corrections and merged.

**Pull request merged**